### PR TITLE
Validation for signup page

### PR DESCRIPTION
**Pull request by RazanTayyem**
Validate signup data:
- [x] Username should be not null and not exist the same username.
- [x] Password should be at least 6-letters.
- [x] Validate the PostCode.

### CODE REVIEW

**Review by ameentalahmeh — COMMENTED** [review]
good :+1:

**Review by mhmdtshref — CHANGES_REQUESTED** [review]
Try to use `sequelize` validations, if there are no validations using sequelize use `Joi`
Also, the validation function should be in the `src/validations` folder files.

File: src/models/Student.model.js [hunk 1]
```diff
@@ -1,32 +1,65 @@
...
+      allowNull: false,
+      defaultValue: false,
+      validate: {
+        isBoolean: true,
+      },
+    },
+    english: {
+      type: Sequelize.BOOLEAN,
+      allowNull: false,
+      defaultValue: false,
```
Comment (mhmdtshref) [inline src/models/Student.model.js]: I don't think that you need to set a default value if `allowNull` is false! Remove it from all these fields.

**Review by ghassanmas — CHANGES_REQUESTED** [review]
Password length could be less than 6

**Review by ghassanmas — APPROVED** [review]

**Pull request merged**